### ISSUE TITLE
Should I change the learning rate according to different GPU nums?

### ISSUE THREAD

**Issue by xzxedu** [biubug6/Pytorch_Retinaface#137]
Hi, many thanks for your work! May I ask will the training setting learning rate be changed automatically when I set different gpu number via CUDA_VISIBLE_DEVICES and ngpu in data/config.py? Is there anything else I need to change?

**Comment by S201961152** [comment]
Hello，I also want  to know this question，Do you solve it？

**Comment by xzxedu** [comment]
You don't need to change the learning rate if you don't change the batch size.

**Comment by S201961152** [comment]
I changed the batchsize，and then how can I change the learning rate？         
            
                
                    
                            
                                
                            
                            
                                1663678924
                            
                    
                        
                            
                                    邮箱：1663678924@qq.com
                            
                        
                
            
        On 12/21/2020 23:23, Zixiao Xing wrote: 
You don't need to change the learning rate if you don't change the batch size.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/biubug6/Pytorch_Retinaface/issues/137#issuecomment-749029313",
"url": "https://github.com/biubug6/Pytorch_Retinaface/issues/137#issuecomment-749029313",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
